test/Controller/AuthenticationTest.php

```diff
@@ -303,11 +303,12 @@ public function it_lets_you_log_out()
             )
         );
 
-        $crawler = $client->request('GET', '/');
+        $crawler = $client->request('GET', '/about');
```

Comment:
The `readyHomePage()` call above would normally be moved below now (since it's now called by this request), but it does mean not automatically redirecting to trigger it at the right point so ok to leave it where it is.

(Not entirely sure why there's an annotations call though. 🤷‍♂)